tensorflow/lite/python/lite.py

```diff
@@ -2407,7 +2407,7 @@ def _validate_quantized_input_stats(self, converter_kwargs, quant_mode):
       raise ValueError(
           "The `quantized_input_stats` flag must be defined when either "
           "`inference_type` flag or `inference_input_type` flag is set to "
-          "tf.int8 or tf.uint8. Currently, `inference_type={}` and "
+          "tf.int8 or tf.uint8. Currently `inference_type={}` and "
```

Comment:
```suggestion
          "tf.int8 or tf.uint8. Currently, `inference_type={}` and "
```

Comma was correct here, removing it causes tests to fail